modules/ycc_vm.py

```diff
@@ -288,7 +287,7 @@
 """
 
 VMS_STATES = ["present", "absent"]
-VMS_OPERATIONS = ["start", "stop", "get_info", "update"]
+VMS_OPERATIONS = ["start", "stop", "get_info", "get_subnet_info", "update"]
```

Comment:
Please mention new operation in [docs](https://github.com/arenadata/ansible-module-yandex-cloud/blob/aa9716486e717d293fdcd5f7423efa967e05b4b6/modules/ycc_vm.py#L200) on top and in [readme](https://github.com/arenadata/ansible-module-yandex-cloud/blob/master/README.md#vm-managment).

Reply:
Please update information [here](https://github.com/arenadata/ansible-module-yandex-cloud/blob/aa9716486e717d293fdcd5f7423efa967e05b4b6/README.md?plain=1#L105).